account_cancel_with_reversal/models/account_invoice.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields,  api, _
+from openerp.exceptions import Warning
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    cancel_move_id = fields.Many2one(
+        'account.move',
+        'Cancelled Journal Entry',
+        copy=False
+        )
```

Comment:
Just for our coding standard, it should have "," at the end of "copy", string for string, and shift left the last bracket. I..e,

cancel_move_id = fields.Many2one(
    'account.move',
    string='Cancelled Journal Entry',
    copy=False,
)